<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomRegExp</class>
 <widget class="QDialog" name="CustomRegExp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" rowspan="3">
     <widget class="QTextBrowser" name="tBFoundSubStr"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lESetRegExp"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTextBrowser" name="tBCurrentHtml">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pBFindRegExp">
        <property name="text">
         <string>Find sub str</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBClear</sender>
   <signal>clicked()</signal>
   <receiver>tBFoundSubStr</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
